_docs/dev-setup.md

```diff
@@ -10,30 +33,6 @@ Some of this may already be familiar for you. Feel free to skip around to the pa
 
 You may ask: why is the website so complicated to set up! You may have a point, but it allows us to write features that let you update the website **without writing code**: we've written templates that generate pages, content, and complicated data relationships, so you don't have to! Hopefully, with practice, you'll find this to be not too tricky.
 
-## Table of Contents
```

Comment:
I think having the table of contents might still be important, but I'll leave this decision to you. If you want a quicker way of generating the table of contents, you can do so with `{:toc}` ([see here for more](https://kramdown.gettalong.org/converter/html.html#toc)).